File: tests/nla/Test1.java
public class Test1 {
     public static void vtrace1(int x, int y, int z, int b, int c){}
     public static void main (String[] args){}

     public static int mainQ(int x){
	  int y = 0;
	  int z = 0;
	  int c = 0;
	  int b = 0;
	  
	  while(true){
	       vtrace1(x, y, z, b, c);
	       if (!(c < x)) break;
    
	       c = c + 1;
	       y = y + 1;
	       if (x == 7) z = z+1;
	       z = z + 1;
	  }
	  return x;
     }
}

File: examples/test1.c
#include <stdio.h>
#include <stdlib.h>

void vassume(int b){}
//void vtrace0(int x, int y){}
void vtrace1(int x, int y){}
//void vtrace2(int q, int r, int a, int b, int x, int y){}
void vtrace2(int x, int y, int q, int r){}

int mainQ(int x, int y){
    vassume(x >= 1 && y >= 1);
    //vtrace0(x,y);  //preconditions

    while(1){
        if (!(x <= 10000))
            break;
        vtrace1(x,y);
        //1<=x <= 10000
        //1<=y
        // 1. -x <= -1
        // 2. -y <= -1
        
        x++;
    }
    
    //vtrace2(x, y, q, r);  //postconditions
    return x;
}

void main(int argc, char **argv){
    mainQ(atoi(argv[1]), atoi(argv[2]));
}




